clients/python/girder_client/__init__.py

```diff
@@ -168,8 +168,29 @@ class GirderClient(object):
 
     DEFAULT_API_ROOT = 'api/v1'
     DEFAULT_HOST = 'localhost'
-    DEFAULT_PORT = 443
-    DEFAULT_SCHEME = 'https'
+    DEFAULT_HTTP_PORT = 8080
```

Comment:
Eh. Seems like it should be 8080, so that people using this with their own local Girder instances find that it "just works". Since DKC uses HTTPS, this works by default with DKC, right?